src/it/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepoSpec.scala

```diff
@@ -153,6 +154,35 @@ class RoutesPostgresRepoSpec extends FunSpec with BeforeAndAfter with Matchers w
             routes.map(_.id.get).toSet should be(Set(3))
           }
         }
+
+        describe("#selectDeletedBefore") {
+          it("should select nothing if there are no deleted routes") {
+            insertRoute("R1")
+            insertRoute("R2")
+
+            val dateTime: LocalDateTime = LocalDateTime.now().plusHours(1L)
+            val routes: List[RouteRow] = routesRepo.selectDeletedBefore(dateTime)
+
+            routes.size shouldBe 0
```

Comment:
But I must admit that the whole assertion topic is messed up. RoutesServiceSpec also uses ```shouldBe``` as a single word. A lot of tests are using ```shouldBe(...)``` as a method.